2001-3000/LIT2842RepCh125.xml

```diff
@@ -36,6 +36,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract>
+            <p>This <foreign xml:lang="gez">malkǝʾ</foreign> was possibly written by an otherwise unknown 
+               Ləssāna Maṣḥaf (see <bibl>
+                  <ptr target="bm:Getatchew2018Cross"/>
+                  <citedRange unit="page">53</citedRange>
+               </bibl>). 
```

Comment:
I don't have Getatchew Haile's book at hand, so I can't check the source, but are we sure that in that context Ləssāna Maṣḥaf was not simply meaning "language of the book", i.e. Gəʿəz?
What about rephrasing as follows: `According to <bibl><ptr target="bm:Getatchew2018Cross"/><citedRange unit="page">53</citedRange></bibl>, this <foreign xml:lang="gez">malkǝʾ</foreign> was possibly written by an otherwise unknown Ləssāna Maṣḥaf`